### PR TITLE
<SelectableAccordion/>- reduce Spacing and adding new verticalPadding prop 

### PR DESCRIPTION
**Pull request by TalZa1810**
### 🔦 Summary
Reduce Spacing- `<SelectableAccordion/>` 

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [x] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [x] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6479

File: src/SelectableAccordion/index.d.ts [hunk 1]
```diff
@@ -9,6 +9,7 @@ export interface SelectableAccordionProps {
   dataHook?: string;
   className?: string;
   type?: 'radio' | 'checkbox' | 'toggle';
+  size?: 'medium' | 'small' | 'tiny';
```
Comment (TalZa1810) [inline src/SelectableAccordion/index.d.ts:12]:
```suggestion
  verticalPadding?: 'medium' | 'small' | 'tiny';
```

File: src/SelectableAccordion/test/SelectableAccordion.tsx [hunk 2]
```diff
@@ -24,6 +24,7 @@ function selectableAccordionWithAllProps() {
         },
       ]}
       type="checkbox"
+      size='small'
```
Comment (TalZa1810) [inline src/SelectableAccordion/test/SelectableAccordion.tsx:27]:
```suggestion
      verticalPadding='small'
```

**Comment by TalZa1810** [comment]
@justinara I've renamed the prop as we discussed

**Review by justinara — APPROVED** [review]

**Comment by TalZa1810** [comment]
fails on a visual which is not related (MultiSelect)

**Pull request merged**